jcl/jpp_configuration.xml

```diff
@@ -73,6 +73,18 @@
 		<parameter name="jxerules:outputdir" value="java/lang"/>
 	</configuration>
 
+	<configuration
+		  label="SIDECAR18-SE-CLOSEDJ9"
+		  outputpath="SIDECAR18-SE/src"
+		  flags="Sidecar18-SE,Sidecar70_27,Sidecar17,Sidecar16,SharedClasses"
+		  jdkcompliance="1.8">
+		<classpathentry kind="lib" path="/binaries/vm/third/rt-compressed.sun180.jar"/>
+		<source path="src/java.base/share/classes"/>
+		<parameter name="macro:define" value="JAVA_SPEC_VERSION=8"/>
+		<parameter name="msg:outputdir" value="com/ibm/oti/util"/>
+		<parameter name="jxerules:outputdir" value="java/lang"/>
+	</configuration>
+
```

Comment:
Suggest this should be a clone of `SIDECAR18-SE-OPENJ9`, except without the flag `Sidecar18-SE-OpenJ9`. I think that would eliminate the need for any of the other changes here.

Reply:
> I think that would eliminate the need for any of the other changes here.

Do you mean all the changes for COMMON_JCL8? The build with OpenJDK needs to have the MethodHandle stubs that are normally only included in an OpenJ9 build, plus the method in ClassLoader, and the z/OS specific code excluded, otherwise it doesn't compile.